src/js/components/sidebar.jsx

```diff
@@ -327,6 +391,21 @@ export default class SideBar extends React.Component {
     browser.tabs.remove(this._getSelectedTabIds());
   }
 
+  _reloadSelected() {
+    const selectedTabIds = this._getSelectedTabIds();
+
+    for (const id of selectedTabIds) {
+      // reload each selected tab.
```

Comment:
Unnecessary comment. It is better to explain *why* you are doing something rather than *what* you are doing. I believe this code is self-documenting, i.e., it does not need an explanation of what it is doing (especially given the context of the `_reloadSelected` method :) )